### ISSUE TITLE
reading from standard input instead of command line argument

### ISSUE THREAD

**Issue by yukinying** [GoogleCloudPlatform/terraform-validator#78]
This would faciliate the use of pipe with terraform-validator. Consider the case for terraform show, this would remove the step of storing the intermediate artifacts on local disk.

**Comment by morgante** [comment]
Yes, this is a good idea. I'd say that if `-` is provided as the plan file name we should read from stdin.

`terraform show -json ./terraform.tfplan | terraform-validator validate --policy-path=${POLICY_PATH} -`
